GNUmakefile

```
@@ -432,7 +432,7 @@ dist_libstemmer_python: $(PYTHON_SOURCES)
 	cp -a $(PYTHON_SAMPLE_SOURCES) $${dest}/src/$(python_sample_dir) && \
 	cp -a $(PYTHON_RUNTIME_SOURCES) $${dest}/src/$(python_runtime_dir) && \
 	cp -a $(PYTHON_PACKAGE_FILES) $${dest} && \
-	(cd $${dest} && $(python) setup.py sdist && cp dist/*.tar.gz ..) && \
+	(cd $${dest} && $(python) setup.py sdist bdist_wheel && cp dist/*.{tar.gz,whl} ..) && \
```

The syntax with curly brackets is a bashism. It won't work when make uses another shell (e.g. dash).

```suggestion
	(cd $${dest} && $(python) setup.py sdist bdist_wheel && cp dist/*.tar.gz dist/*.whl ..) && \
```